guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -340,8 +340,8 @@ $ TEST_DIR=generators bundle exec rake test
 You can run the tests for a particular file by using:
 
 ```bash
-$ cd actionpack
-$ bundle exec ruby -w -Itest test/template/form_helper_test.rb
+$ cd actionview
```

Comment:
I changed this because the test file is actually located in `actionview`. The original commands will fail.